nano/nano_node/entry.cpp

```diff
@@ -78,9 +79,18 @@ int main (int argc, char * const * argv)
 		}
 	}
 
+	size_t batch_size;
+	if (vm.count ("batch_size"))
+	{
+		batch_size = (vm["batch_size"].as<size_t> ());
```

Comment:
You are ruining my work :P (See #1)
https://github.com/nanocurrency/nano-node/pull/1514